clang/include/clang/Driver/Driver.h

```diff
@@ -684,6 +689,17 @@ class Driver {
   const std::string getFPGATempDepFile(const std::string &FileName) const {
     return FPGATempDepFiles[FileName];
   }
+
+  /// addIntegrationFiles - Add the integration files that will be populated
+  /// by the device compilation and used by the host compile.
+  void addIntegrationFiles(const char *IntHeaderName,
```

Comment:
Should it be called `addIntegrationFile` instead? It doesn't seem to support attaching more than one header to a single source.

Related question: will it work together with recently added "integration footer"? intel/llvm#3455

Reply:
I had early hooks to add the integration footer changes here, but removed them to not clutter the review.  When the integration footer changes are added, the naming of `addIntegrationFiles` makes more sense.

Reply:
Ok, thanks for the explanation